stage1/enter_kvm/enter_kvm.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"syscall"
+
+	"github.com/coreos/rkt/networking/netinfo"
+	"github.com/coreos/rkt/pkg/lock"
+)
+
+const (
+	kvmSettingsDir        = "kvm"
+	kvmPrivateKeyFilename = "ssh_kvm_key"
+)
+
+var (
+	PODPid  string
+	appName string
+)
+
+func init() {
+	flag.StringVar(&PODPid, "pid", "", "podPID")
+	flag.StringVar(&appName, "appname", "", "application to use")
+}
+
+func getPODDefaultIP(workDir string) (string, error) {
+	// get pod lock
+	l, err := lock.NewLock(workDir, lock.Dir)
+	if err != nil {
+		return "", err
+	}
+
+	// get file descriptor for lock
+	fd, err := l.Fd()
+	if err != nil {
+		return "", err
+	}
+
+	// use this descriptor as method of reading pod network configuration
+	nets, err := netinfo.LoadAt(fd)
+	if err != nil {
+		return "", err
+	}
+	// kvm flavored container must have at first position default vm<->host network
+	if len(nets) == 0 {
+		return "", fmt.Errorf("POD has no configured networks")
+	}
+	return nets[0].IP.String(), nil
+}
+
+func execToSSH() int {
```

Comment:
I don't like "int as error code return here" - we should stick with error created by errors.Errorf() and then in main()
```go
err := execToSSH()
if err != nil {
   fmt.Fprintf(os.Stderr, "execToSSH: %v\n", err)
   os.Exit(1)
}
```
the resposbility how to handle/display error should be left to caller